infrastructure/docker/build/Dockerfile-tsb

```diff
@@ -0,0 +1,40 @@
+# Licensed to the Apache Software Foundation (ASF) under one
+# or more contributor license agreements.  See the NOTICE file
+# distributed with this work for additional information
+# regarding copyright ownership.  The ASF licenses this file
+# to you under the Apache License, Version 2.0 (the
+# "License"); you may not use this file except in compliance
+# with the License.  You may obtain a copy of the License at
+#
+#   http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing,
+# software distributed under the License is distributed on an
+# "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+# KIND, either express or implied.  See the License for the
+# specific language governing permissions and limitations
+# under the License.
+
+ARG RHEL_VERSION=8
+FROM centos:${RHEL_VERSION} as tsb-build
+RUN yum clean all && yum -y install git
+COPY GO_VERSION /
+RUN curl -L https://dl.google.com/go/go$(cat /GO_VERSION).linux-amd64.tar.gz | tar -C /usr/local -xz && ln -s /usr/local/go/bin/go /usr/bin/go
+RUN mkdir -p /go/src/github.com/comcast/tsb && cd /go/src/github.com/comcast/tsb && git init && git remote add origin https://github.com/comcast/tsb && git fetch --depth=1 origin 851be3e137b20ba5d2d1eefd07cd303dc5f53865 && git reset --hard FETCH_HEAD
```

Comment:
> 851be3e137b20ba5d2d1eefd07cd303dc5f53865
Please tag this commit and use the tag instead

Reply:
Tags can change, commit hashes cannot. I believe the commit hash is the less ambiguous mechanism.

Reply:
That is an argument against all git tags. Please use a git tag

Reply:
I'm with @alficles, I think the actual commit is safer and more accurate here.
Also, this is a commit in github.com/comcast/tsb, unless I'm mistaken? Apache Traffic Control doesn't really have authority to demand changes to other repos (some individuals here have the influence in this particular case, but that isn't always going to be true).

Reply:
> I'm with @alficles, I think the actual commit is safer and more accurate here.
> Also, this is a commit in github.com/comcast/tsb, unless I'm mistaken? Apache Traffic Control doesn't really have authority to demand changes to other repos (some individuals here have the influence in this particular case, but that isn't always going to be true).

Fair enough, there's no need to add a tag, I just assumed he would prefer a tag to a branch. But using a branch and checking the commit hash is fine, too.

Using a git tag or git branch and verifying the commit hash is no less secure than using a commit hash directly, yet it helps human-readability immensely.

Reply:
I have added a comment in the latest commit.